### DISCUSSION TITLE
Color=Color.Custom

### DISCUSSION THREAD

**Discussion by MazzGuille** [microsoft/fluentui-blazor#1608] [Q&A] [answered]
I would like to know how to configure a specific color using this, if I hover over custom it says it receives a string with RGB values, see image for reference, but I don't know how to add that, can I please get some help?

![image](https://github.com/microsoft/fluentui-blazor/assets/97600586/4e517fde-02cd-43eb-b779-e9c9643a6347)

**Comment by vnbaaij** [comment, marked as answer]
It is exactly what it says in the tooltip. Add a parameter CustomColor to the icon component with a hex value in one of the 2 formats

**Reply by MazzGuille** [reply]
Im sorry, I am kind of lost, I understand I should add it in a () but if I do that, I get an error, would you be so kind to share an image or some text of how it should be done, thanks

**Reply by vnbaaij** [reply]
See the Razor tab at https://fluentui-blazor.net/Icon#coloroptions

**Reply by MazzGuille** [reply]
Thanks!

**Comment by LocalStokell** [comment]
The FluentNavLink allows for custom color parameter to be set. 

`<FluentNavLink Icon="@(new Microsoft.FluentUI.AspNetCore.Components.Icons.Regular.Size24.History())" Color="@Color.Custom" CustomColor="#87CEEB" Href="" Class="nav-item nav-link">History</FluentNavLink>`

But then when done you just get said error

<img width="918" height="142" alt="image" src="https://github.com/user-attachments/assets/69409a8b-d290-4a54-a5e4-e40ca9d6f2a9" />

**Reply by vnbaaij** [reply]
You need to use the `IconColor` parameter in this case
